packages/dx-react-grid/docs/reference/table-header-row.md

```diff
@@ -20,6 +20,7 @@ Name | Type | Default | Description
 -----|------|---------|------------
 cellComponent | ComponentType&lt;[TableHeaderRow.CellProps](#tableheaderrowcellprops)&gt; | | A component that renders a header cell.
 rowComponent | ComponentType&lt;[Table.RowProps](table.md#tablerowprops)&gt; | | A component that renders a header row.
+sortingComponent | ComponentType&lt;[TableHeaderRow.SortingControlProps](#tableheaderrowsortingcontrolprops)&gt; | | A component that renders a sorting control.
```

Comment:
I don't like the name. Maybe we can use something like SortLabel here?

Reply:
I don't mind. @viterobk what do you think?

Reply:
Label does not imply interaction. May be SortingControl or SortingSwitch?